resources/assets/coffee/osu_common.coffee

```diff
@@ -235,6 +235,26 @@
   transChoice: (key, count, replacements) ->
     message = Lang.choice key, count, replacements, currentLocale
 
+  uuid: ->
+    Turbolinks.uuid() # no point rolling our own
+
+  updateQueryString: (key, value, url = window.location.href) ->
+    regExp = new RegExp("([?&])" + key + "=.*?(&|#|$)(.*)", "gi")
```

Comment:
technically should be `_.escapeRegExp(decodeURIComponent(key))`? Not sure if `i` flag is needed. Also, string interpolation.